### PR TITLE
 WEB-Email-troubleshooting-Lock_for_SPAM-SK-1548

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

File: pages/web_cloud/email_and_collaborative_solutions/troubleshooting/locked_for_spam/guide.fr-fr.md [hunk 1]
```diff
@@ -27,6 +27,16 @@ Avant de poursuivre et si le blocage concerne une adresse e-mail de type MXplan,
 
 Lorsqu'une activité suspecte est détectée au niveau de l'envoi des e-mails, l'adresse concernée est automatiquement bloquée. Dans cette situation, vous ne pouvez plus envoyer d'e-mails depuis cette adresse e-mail.
 
+> [!warning]
+>
+> Une « activité suspecte » signifie que :
+>
+> - Le serveur anti-spam, qui scanne les e-mails à l'envoi, a constaté qu'un ou plusieurs éléments de l'e-mail sont considérés comme suspectes et peuvent constituer un e-mail SPAM.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/troubleshooting/locked_for_spam/guide.fr-fr.md]:
```suggestion
> - Le serveur anti-spam, qui scanne les e-mails à l'envoi, a constaté qu'un ou plusieurs éléments de l'e-mail sont considérés comme suspects et peuvent constituer un e-mail de SPAM.
```

File: pages/web_cloud/email_and_collaborative_solutions/troubleshooting/locked_for_spam/guide.fr-fr.md [hunk 2]
```diff
@@ -27,6 +27,16 @@ Avant de poursuivre et si le blocage concerne une adresse e-mail de type MXplan,
...
+> - La fréquence d'envoi et le nombre de destinataires sont trop importants et contribuent à considérer l'envoi comme du spamming. En effet pour réaliser des envoies massif, il est nécessaire d'utiliser un service de mailing list mais pas une adresse e-mail standard.
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/troubleshooting/locked_for_spam/guide.fr-fr.md]:
```suggestion
> - La fréquence d'envoi et le nombre de destinataires sont trop importants et contribuent à considérer l'envoi comme du spamming. En effet, pour réaliser des envois massifs, il est nécessaire d'utiliser un service de mailing list et non une adresse e-mail standard.
```

File: pages/web_cloud/email_and_collaborative_solutions/troubleshooting/locked_for_spam/guide.fr-fr.md [hunk 3]
```diff
@@ -27,6 +27,16 @@ Avant de poursuivre et si le blocage concerne une adresse e-mail de type MXplan,
...
+>
+> Les raisons précises d'un blocage ne peuvent pas être divulguées pour éviter toute tentative de contournement du système de détection des SPAM. Pour tester le contenu d'un e-mail, vous pouvez utiliser un outil externe à OVHcloud comme [Mailtester](https://www.mail-tester.com/).
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/email_and_collaborative_solutions/troubleshooting/locked_for_spam/guide.fr-fr.md]:
```suggestion
> Les raisons précises d'un blocage ne peuvent pas être divulguées pour éviter toute tentative de contournement du système de détection des SPAM. Pour tester le contenu d'un e-mail, vous pouvez utiliser un outil externe à OVHcloud tel que [Mailtester](https://www.mail-tester.com/){.external}.
```

**Comment by Kazarus4** [comment]
After @OVH-Team-Guides "suggestion" commits, all are ok for me.

**Review by Kazarus4 — APPROVED** [review]

**Comment by BrenKt** [comment]
Translation check to do please

**Review by bs-ovh — APPROVED** [review]

**Pull request merged**